CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # node-google-dfp-wrapper
 
+### 1.3.0
+- Adds `getCriteriaValues` for querying criteria values without extracting id.
+- Adds `getCriteriaValueId` for querying criteria values and extracting id.
```

Comment:
So to clarify:
 `getCriteriaValues` is for querying multiple criteria values.
 `getCriteriaValueId` is for querying an id of a criteria value.
 `getCriteriaValue` is for querying an id of a criteria value.

Is that right?